Sources/SpeziAccount/Resources/Localizable.xcstrings

```diff
@@ -0,0 +1,2090 @@
+            "value" : "Identidad de Género"
+          }
+        }
+      }
+    },
+    "GENDER_IDENTITY_TRANSGENDER" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "de" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Transgender"
+          }
+        },
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Transgender"
+          }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Transgénero"
+          }
+        }
+      }
+    },
+    "General Settings" : {
+
+    },
+    "Hello" : {
+
+    },
+    "Hello World" : {
+
+    },
+    "License Information" : {
+
+    },
+    "MISSING_ACCOUNT_DETAILS" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "de" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "**Kein Benutzerkonto gefunden.**\\n\\nDiese Oberfläche erfordert ein aktives Benutzerkonto.\\nBitte kontaktiere die Dokumentation für die AccountSetup Oberfläche, um ein Benutzerkonto einzurichten."
+          }
+        },
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "**Couldn't find a user account.**\\n\\nThis view requires an active user account.\\nRefer to the documentation of the AccountSetup view on how to setup a user account!"
+          }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "**No se pudo encontrar una cuenta de usuario.**\\n\\nEsta pagina require una cuenta de usuario activa.\\n¡Consulte la documentación de la pagina 'AccountSetup' sobre cómo configurar una nueva cuenta de usuario!"
+          }
+        }
+      }
+    },
+    "MISSING_ACCOUNT_SERVICES" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "de" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "**Es wurden keine Account Services konfiguriert.**\\n\\nBitte kontaktiere die Dokumentation von SpeziAccount für mehr Informationen zur Konfiguration eines AccountServices!"
+          }
+        },
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "**No Account Services set up.**\\n\\nPlease refer to the documentation of the SpeziAccount package on how to set up an AccountService!"
+          }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "**No hay servicios de cuenta configurados.**\\n\\n¡Por favor, consulte la documentación del paquete SpeziAccount sobre cómo configurar un servicio de cuenta!"
+          }
+        }
+      }
+    },
+    "Mock AccountService" : {
+
+    },
+    "Mock SignIn with Apple" : {
+
+    },
+    "Mock Simple AccountService" : {
+
+    },
```

Comment:
Yep, seems like I forgot to use the `Text(verbatim:)` initializer all the Mock implementations.